packages/web-components/src/components/card-group/card-group.ts

```diff
@@ -166,6 +166,14 @@ class DDSCardGroup extends LitElement {
   @property({ attribute: 'grid-mode', reflect: true })
   gridMode = GRID_MODE.COLLAPSED;
 
+  /**
+   * An optional 1px border surrounding the component.
+   * Hidden/0px (default) | Visible/1px).
+   */
+
+  @property({ attribute: 'border', reflect: true })
+  optionalBorder = OPTIONAL_BORDER.HIDDEN;
```

Comment:
Just a little nitpick, I think it would be easier for adopters to only have to pass in a boolean for enabling the border. So instead of having to use border='visible' they can simply pass in border to enable, or omit to have the default borderless group.

```suggestion
  @property({ type: Boolean, reflect: true })
  border = false;
```

Reply:
Great suggestion, just committed the changes 👍